cmd/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"google.golang.org/grpc"
+	"golang.org/x/net/context"
+	"github.com/heroku/silvia-runtime-university/client"
+	"github.com/heroku/silvia-runtime-university/spec"
+	"github.com/joeshaw/envdecode"
+)
+
+func main() {
+	flag.String("help", "", "Display usage")
```

Comment:
You can actually omit this line; `flag.Parse()` automatically handles the help flag! If you added some more flags, you'd see the flags documented when you run your program with `-help`.

One idea would be to support a `-server-url` flag that's used instead of SERVER_URL if it's present.